README.md

```diff
@@ -7,11 +7,22 @@
 
 ### _Instagram_, dummified
 
-![Issues](https://img.shields.io/github/issues/narayan954/dummygram?color=brightgreen)
-![Pull requests](https://img.shields.io/github/issues-pr/narayan954/dummygram)
-![Forks](https://img.shields.io/github/forks/narayan954/dummygram)
-![Stars](https://img.shields.io/github/stars/narayan954/dummygram)
-![Licence](https://img.shields.io/github/license/narayan954/dummygram?color=orange)
+<div align="center">
+      <img src="https://forthebadge.com/images/badges/built-with-love.svg" />
+      <img src="https://forthebadge.com/images/badges/uses-brains.svg" />
+      <img src="https://forthebadge.com/images/badges/powered-by-responsibility.svg" />
+      <br>
+      <img src="https://img.shields.io/github/repo-size/DevFeed404/DevFeed-1.0?style=for-the-badge" />
```

Comment:
update these metrics for our repo.(provide our repo name as narayan954/dummygram)